src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServiceCommands.cs

```diff
@@ -84,6 +84,8 @@ private void StartServiceCallback(object sender, EventArgs e)
         {
             if (_startMenuCmd.Enabled)
             {
+                IntegrationTestTraceListener.Install();
```

Comment:
For unit tests we generally setup the trace listeners with app.config. We could just use a diff file for the integration tests.

https://github.com/dotnet/roslyn/blob/master/eng/config/test/Desktop/app.config#L9

Reply:
How would this get applied to devenv.exe?

Reply:
Ah I see. Yeah that's not gonna work. I thought of this as inside the xunit process but it's in devenv.exe.